e2e/test-component/scenarios/embedding-sdk/editable-dashboard.cy.spec.tsx

```diff
@@ -447,4 +450,47 @@ describe("scenarios > embedding-sdk > editable-dashboard", () => {
       });
     });
   });
+
+  describe("subscriptions", () => {
+    beforeEach(() => {
+      cy.signInAsAdmin();
+      H.setupSMTP();
+      const questionCard: Partial<DashboardCard> = {
+        id: ORDERS_DASHBOARD_DASHCARD_ID,
+        card_id: ORDERS_QUESTION_ID,
+        row: 0,
+        col: 0,
+        size_x: 16,
+        size_y: 8,
+      };
+      H.createDashboard({
+        name: DASHBOARD_NAME,
+        dashcards: [questionCard],
```

Comment:
interactive embedding test suite's dashboard `@dashboardId` doesn't have any data card, and we need one to use subscriptions.